### ISSUE TITLE
Install on android running via Waydroid?

### ISSUE THREAD

**Issue by gideion** [yuliskov/SmartTube#3209]
Is it possible to install SmartTube on Waydroid?

Waydroid is a s container-based approach to boot a full Android system on a regular GNU/Linux system like Ubuntu. 
I am currently able to install and run other android apps from the play store as well as F-Droid.

I tried to install SmartTube with no success. I suspect the reason the install failed is due to the app not being compiled for my x86-64 processor. 

I personally have only limited experience compiling my own apps, it has been a few years since i have done so, so i have not yet attempted to compile it myself. 

Or maybe the problem is something else?

Thanks!

**Comment by vansoest** [comment]
+1

**Comment by jesterpaul** [comment]
Yes Please!. Gutted it doesn't install, as it would give me Smart Tube on my laptop

**Comment by khuedoan** [comment]
I was able to use SmartTube on Waydroid with the following setup:

- OS: Bazzite (other distros should work too)
- Waydroid with some extra setup to enable `libndk` (AMD) for ARM translation https://docs.bazzite.gg/Installing_and_Managing_Software/Waydroid_Setup_Guide/

**Comment by khuedoan** [comment]
Oh nice I didn't know that, thank you!
